lib/EarlyFunctionAnalysis/EarlyFunctionAnalysis.cpp

```diff
@@ -602,14 +666,14 @@ finalizeModel(GeneratedCodeBasicInfo &GCBI,
   }
 
   // Finish up the CFG
-  for (const auto &F : Functions) {
-    MetaAddress EntryPC = getBasicBlockPC(F.Entry);
-    model::Function &Function = Binary.Functions[EntryPC];
-    auto &Summary = Properties.at(EntryPC);
+  for (model::Function &Function : Binary.Functions) {
+    if (Function.CFG.size() != 0)
+      continue;
```

Comment:
Why?

Reply:
As much as we want to skip the commit of the prototype to the model if the original function has type different from `model::FunctionType::Invalid`, we also want to the skip the commit of its CFG.

Reply:
Why use a different criterion here?

I think you're checking `Funciton.CFG.size()` because you can no longer rely on `.Type` being `Invalid`, is that right?

However, you can simply create a `set` of `model::Function *` that need to work on and iterate over those. No need to come up with "alternative ways".

Reply:
Sounds good.